config/testdata/single_plugin.toml

```diff
@@ -1,7 +1,9 @@
 [[inputs.memcached]]
   servers = ["localhost"]
   namepass = ["metricname1"]
+  namepass_separator = "."
   namedrop = ["metricname2"]
+  namedrop_separator = "."
   fieldpass = ["some", "strings"]
```

Comment:
Can you please leave existing tests as they were!? We usually use them as a guarantee for backward-compatibility... Same for the other toml files below. If you want to test something, just add a new config or test.

Reply:
makes sense, done